electron/main.ts

```diff
@@ -361,49 +364,52 @@ ipcMain.on('callTool', async (event, { tool, wine, prefix, exe }: Tools) => {
     command = `WINEPREFIX=${winePrefix} ${wineBin} ${exe}`
   }
 
-  console.log({ command })
+  Logger.info({message: `Trying to run command ${command}`, service: 'ipcMain::callTool'})
   return exec(command)
 })
 
 /// IPC handlers begin here.
 
-ipcMain.handle('checkGameUpdates', () => LegendaryGame.checkGameUpdates())
+ipcMain.handle('checkGameUpdates', () => {Logger.info({message: 'Attempting to check for game updates ', service: 'ipcMain::checkGameUpdates'}); return LegendaryGame.checkGameUpdates()})
```

Comment:
Something must've broken here, I'll investigate.

Reply:
I believe it's because I didn't add return statements, and assumed it would just infer the data that was returned by the second function 🤦🏼‍, but slapping some returns in fixed it. However, I think it might be a good idea for me to add this in or the log files will just grow forever on people's machines.
https://www.npmjs.com/package/winston-daily-rotate-file

Reply:
Exactly, when using arrow functions, if you need to have brackets, your must use return. if you're just returning some value, then you don't need the return statement:
`const fn01 = (a) => a * 10; // here you don't need the return because it is a pure function`

On the other hand, here the return is mandatory, otherwise the fn will fail:
```
const fn02 = (a) => { 
  const b = a * 10 
  const c = a + b
  return b + c
}
```